pkg/fission-cli/cmd/environment/pods.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 The Fission Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package environment
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/fission/fission/pkg/fission-cli/cliwrapper/cli"
+	"github.com/fission/fission/pkg/fission-cli/cmd"
+	flagkey "github.com/fission/fission/pkg/fission-cli/flag/key"
+	"github.com/fission/fission/pkg/fission-cli/util"
+)
+
+type ListPodsSubCommand struct {
+	cmd.CommandActioner
+}
+
+func ListPods(input cli.Input) error {
+	return (&ListPodsSubCommand{}).do(input)
+}
+
+func (opts *ListPodsSubCommand) do(input cli.Input) error {
+
+	m := &metav1.ObjectMeta{
+		Name: input.String(flagkey.EnvName),
+		Labels: map[string]string{
+			"environmentNamespace": input.String(flagkey.NamespaceEnvironment),
```

Comment:
We should use constants for query parameters as mentioned above.
These ensure if we change client-side parameters, we are also changing server-side.